Userland/Utilities/hexdump.cpp

```diff
@@ -12,11 +12,19 @@
 
 static constexpr size_t LINE_LENGTH_BYTES = 16;
 
+enum class STATES {
+    PRINT,
+    PRINT_FILLER,
+    NOPRINT
+};
+
 int main(int argc, char** argv)
 {
     Core::ArgsParser args_parser;
     const char* path = nullptr;
+    bool verbose;
     args_parser.add_positional_argument(path, "Input", "input", Core::ArgsParser::Required::No);
+    args_parser.add_option(verbose, "Cause hexdump to display all input data.", "verbose", 'v');
```

Comment:
All of the options "cause hexdump to X", let's just skip that part:
```suggestion
    args_parser.add_option(verbose, "Display all input data", "verbose", 'v');
```

maybe even `"Disable line coalescing"`?